lib/tools.py

```diff
@@ -325,7 +325,7 @@ def sort_csv(filename):
     csv_reader_infile = csv.reader(input_file)
 
     # The first entry is the header line
-    infields = next(csv_reader_infile)
+    infields = ""
```

Comment:
The goal of the original line was to grab the first line and store it for later. As such, it seems like if we're just going to assign `infields` `""` then we can just get rid of the line and associated comment entirely. That said, from a first read it seems as if this just accomplished the same thing as the changes below now accomplish, which is using the first row as the header and not writing it. Is this change addressing one of the Python 2/3 csv differences? If so it should be documented with a comment. If not can you explain what the change is doing? Thanks.